mslib/msui/qt5/ui_about_dialog.py

```diff
@@ -109,7 +110,7 @@ def setupUi(self, AboutMSUIDialog):
         self.verticalLayout.addLayout(self.verticalLayout_2)
 
         self.retranslateUi(AboutMSUIDialog)
-        self.btOK.clicked.connect(AboutMSUIDialog.accept)
+        self.btOK.clicked.connect(AboutMSUIDialog.accept) # type: ignore
```

Comment:
where does this comment come from?

Reply:
I looked at the about dialog, the information is incorrect.


```
Please read the reference documentation:

Rautenhaus, M., Bauer, G., and Doernbrack, A.: A web service based tool to plan
atmospheric research flights, Geosci. Model Dev., 5,55-71, doi:10.5194/gmd-5-55-2012, 2012.

and the paper's Supplement (which includes a tutorial) before using the application. The documents are available at:

 * http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf
 * http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012-supplement.pdf
 * https://gmd.copernicus.org/articles/15/8983/2022/gmd-15-8983-2022.pdf

When using this software, please be so kind and acknowledge its use by citing the above mentioned reference documentation in publications, presentations, reports, etc. that you create. Thank you very much.
```

The new paper ist not a supplement of the existing one. supplement means some additional information.

So it must be changed to. 

```
Please read the reference documentation:

Bauer, R., Grooß, J.-U., Ungermann, J., Bär, M., Geldenhuys, M., and Hoffmann, L.: The Mission Support System (MSS v7.0.4) and its use in planning for the SouthTRAC aircraft campaign, Geosci. Model Dev., 15, 8983–8997, https://doi.org/10.5194/gmd-15-8983-2022, 2022.


Rautenhaus, M., Bauer, G., and Doernbrack, A.: A web service based tool to plan
atmospheric research flights, Geosci. Model Dev., 5,55-71, doi:10.5194/gmd-5-55-2012, 2012.

and the paper's Supplement (which includes a tutorial) before using the application. The documents are available at:

 * http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf
 * http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012-supplement.pdf
 
 When using this software, please be so kind and acknowledge its use by citing the above mentioned reference documentation in publications, presentations, reports, etc. that you create. Thank you very much.
```

Reply:
The font size was changed to 10 from 11 in the new commit, if any changes are required I will do so.

Reply:
I guess we can have the origin 11 font size, because dependent on the window manager there is anyway a scroll region. It is not needed to show that all reduced without scrolling.